### PR TITLE
MARS IPFS 1.1 Goal - CIDs and more

### LINKED ISSUE #270: IPFS Update Sept-Oct 2022 IPFS 1.1
**Describe the use case**
We are working through an update to the learning goals, names of sections, and where we arrange the content in that section. See the work here: https://miro.com/app/board/uXjVOB6tchQ=/

**Describe the problem**
Update the section(s) of the curriculum & learning Goals

**To Dos**
- [x] Update the Goals on the Learning Objectives page of IPFS Curriculum

- [x] Update sections (name if applicable) & come up with sub-goals

- [x] Move and re-write content (without deleting content) to align to learning goals

- [x] Update [Scope & Sequence](https://www.notion.so/pl-strflt/8e6c3e8aa7a446ecb4a2f6cd55462218?v=43c411da4dbf45838a18716547c50929)

- [ ] Update the [Assessment Qs](https://docs.google.com/document/d/1cpVT7EkcWzKe0Vb1ENk_jeSY1tB6RhVzy0fmXZbvao0/edit?usp=sharing)

- [ ] Update the [Pre & section quizzes](https://drive.google.com/drive/u/0/folders/1VUzXRQaYNIO-nR2j5Kut0nkbHnxgELTs)

**When you work on this issue, start on the branch**:
```
git checkout feat/ipfs-update
git checkout -b MAR/ipfs1.1
```

### PR DESCRIPTION
**Pull request by mrodriguez3313**
fixes #270

### CODE REVIEW

**Comment by mrodriguez3313** [comment]
@enoldev ready for review. I added the goals in the json and added some deep tags

File: content/en/curriculum/ipfs/overview-of-ipfs/index.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+- 1.13
+- 1.14
+level:
+- shallow
+- deep
+---
+
+## Objectives
+
+**IPFS 1.1 Understand Content Addressing, CIDs, and what advantages they present over location-based addressing**
```
Comment (enoldev) [inline content/en/curriculum/ipfs/overview-of-ipfs/index.md:23]: You can remove this section, as we're including the goals in the JSON

File: content/en/curriculum/ipfs/overview-of-ipfs/index.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+{{< youtube dWq0GNbLtUI>}}
+
+At a high level, the talk covered the following concepts:
+
+**The Importance of Immutabilty**
+
+IPFS allows us to verify the content we received is the content we asked for using hashes.
+* Hashes are deterministic, meaning given an input, it will always have the same output. If the input data changes in any way, then the output generated hash will also change.
+* Since files are immutable, they can never change without generating a brand new hash and, by extension, CID.
+* IPFS enables fast caching and deduplication, this means you can save space by just communicating any changes to a file.
```
Comment (enoldev) [inline content/en/curriculum/ipfs/overview-of-ipfs/index.md:39]:
```suggestion
* IPFS enables fast caching and deduplication. This means you can save space by just communicating any changes to a file.
```
or
```suggestion
* IPFS enables fast caching and deduplication, which means that you can save space by just communicating any changes to a file.
```

**Review by enoldev — APPROVED** [review]

File: content/en/curriculum/ipfs/overview-of-ipfs/index.md (hunk 1 above)
Comment (mrodriguez3313) [reply content/en/curriculum/ipfs/overview-of-ipfs/index.md:23]: oh yeah lol

**Pull request closed without merging**